charts/eks-anywhere-packages/README.md

```diff
@@ -0,0 +1,65 @@
+# eks-anywhere-packages
+
+A Helm chart for eks-anywhere-packages, a controller for managing the lifecycle of EKS Anywhere packages.
+
+![Version: 0.0.0](https://img.shields.io/badge/Version-0.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.0.0](https://img.shields.io/badge/AppVersion-0.0.0-informational?style=flat-square)
+
+## Installing the Chart
+
+Before the chart can be installed the we need to install cert-manager.
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+```
+
+To do a simple install run:
+
+```bash
+helm upgrade --install --namespace eksa-packages --create-namespace \
+  eksa-packages eks-anywhere-packages/ \
+  --values values.yaml
```

Comment:
I would expect the chart to create and use the namespace and the namespace to be overridden possibly in values.yaml

Reply:
You can override the namespace in the values actually, but helm isn't good at creating the namespace via a template so most people do it this way, or create the namespace beforehand with a command.

Reply:
does the order of the namespace and create-namespace flag matter? If not, we could provide create-namespace first, then namespace after to make the creation clear?